### PR TITLE
Add Theme "wholespace"

### PR DESCRIPTION
**Pull request by wholespace**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

### Description

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

wholespace

### CODE REVIEW

File: themes/wholespace.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "alignment": "left",
+      "newline": true,
```
Comment (JanDeDobbeleer) [inline themes/wholespace.json]: why start with a newline?

File: themes/wholespace.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          "style": "plain",
+          "template": " \uf7d4 ",
+          "type": "exit"
+        }
+      ],
+      "type": "prompt"
+    }
+  ],
+  "console_title_template": "{{ .Folder }}",
+  "osc99": true,
```
Comment (JanDeDobbeleer) [inline themes/wholespace.json]: this has been replaced, please use `"pwd": "osc99"`, BUT ideally remove it as it's very terminal specific.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
A few remark, please have a look

File: themes/wholespace.json (hunk 1 above)
Comment (wholespace) [reply themes/wholespace.json]: I have done fix.
Thank you

File: themes/wholespace.json (hunk 2 above)
Comment (wholespace) [reply themes/wholespace.json]: I have done fix.
Thank you

**Pull request merged**